src/main/resources/api.yaml

```diff
@@ -87,7 +87,115 @@ paths:
           description: User is not authorized
         404:
           description: Project not found
-  /projects/{id}/sprints:
+  /projects/{projectId}/members:
+    get:
+      tags:
+        - projects
+      summary: Retrieves users belonging to the project
+      operationId: getUsersFromProject
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+      responses:
+        200:
+          description: Users retrieved successfully
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/UserInProjectDto'
+        404:
+          description: Project not found
+    post:
+      tags:
+        - projects
+      summary: Adds a user to the project
+      operationId: addUserToProject
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: projectId
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+        - in: body
+          name: request
+          description: The user's e-mail
+          required: true
+          schema:
+            type: string
+      responses:
```

Comment:
Tutaj jakby co idea jest taka, że ten user ma już istnieć - w sensie nie następuje bieda-rejestracja automatycznie, tylko jak nie ma kogoś z takim mailem, to ups

Reply:
A tak nie jest? :open_mouth:

Reply:
A nie wiem, nie wiem - tak się tylko upewniam